app/Legacy/Models/PlayerBadge.php

```diff
@@ -5,8 +5,85 @@
 namespace App\Legacy\Models;
 
 use App\Legacy\Database\Eloquent\BaseModel;
+use RA\AwardType;
 
 class PlayerBadge extends BaseModel
 {
+    // Note: will be renamed and split into Community/UserBadge and Platform/PlayerBadge
```

Comment:
It has yet to be decided if a split, of what currently is known as "SiteAwards", into two distinct types of Badges is applicable. The addition of `Community/Models/UserBadge` to the existing `Platform/Models/PlayerBadge` is a mere preparation for that for now.